spec/javascript/components/supplemental_application/sections/ConfirmedHousholdIncome.test.js

```diff
@@ -58,11 +58,11 @@ describe('ConfirmedHouseholdIncome', () => {
     const mockAmiCharts = [
       {
         ami_chart_type: 'chart1',
-        ami_chart_year: 2020
+        ami_chart_year: 2021
```

Comment:
I thinnk after your rebase these changes will break this test since I added `currentYearOverride={2020}` to the getWrapper function in this file.